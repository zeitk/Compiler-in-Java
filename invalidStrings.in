"Unterminated string
"
"Unterminated string \"
"Unterminated \\ string \"
"Bad escape \\ character \ "
"Bad escape \l  character"
"\d Bad escape character \""
"\q"
"Unterminated and bad escape character \p 
"Unterminated \" \ and bad escape character \"
"\! Unterminated and bad escape character
"\ 
"Unterminated and bad escape character \\ \" \u
identifier"
