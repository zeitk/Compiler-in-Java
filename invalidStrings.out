1:1 ****ERROR**** ignoring unterminated string literal
2:1 ****ERROR**** ignoring unterminated string literal
3:1 ****ERROR**** ignoring unterminated string literal
4:1 ****ERROR**** ignoring unterminated string literal
5:1 ****ERROR**** ignoring string literal with bad escaped character
6:1 ****ERROR**** ignoring string literal with bad escaped character
7:1 ****ERROR**** ignoring string literal with bad escaped character
8:1 ****ERROR**** ignoring string literal with bad escaped character
9:1 ****ERROR**** ignoring unterminated string literal with bad escaped character
10:1 ****ERROR**** ignoring unterminated string literal with bad escaped character
11:1 ****ERROR**** ignoring unterminated string literal with bad escaped character
12:1 ****ERROR**** ignoring unterminated string literal with bad escaped character
13:1 ****ERROR**** ignoring unterminated string literal with bad escaped character
Error: Non-string token returned
Error: Character count not reset correctly
14:11 ****ERROR**** ignoring unterminated string literal
