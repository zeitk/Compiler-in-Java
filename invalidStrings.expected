ignoring unterminated string literal
ignoring unterminated string literal
ignoring unterminated string literal
ignoring unterminated string literal
ignoring string literal with bad escaped character
ignoring string literal with bad escaped character
ignoring string literal with bad escaped character
ignoring string literal with bad escaped character
ignoring unterminated string literal with bad escaped character
ignoring unterminated string literal with bad escaped character
ignoring unterminated string literal with bad escaped character
ignoring unterminated string literal with bad escaped character
ignoring unterminated string literal with bad escaped character
